### PR TITLE
Add Windows Build Instructions

### PR DESCRIPTION
**Pull request by bjoekeldude**
Please test this on other machines before merging!

### CODE REVIEW

**Comment by R-Abbasi** [comment]
- Supposedly a green tick mark is better than a red X one for the components needed to install.
- `Microsoft PowerShell` is likely required too. 
- I'd also suggest removing `--config Release` at least for the development stage. 

It's also suitable to use _vs-code_ - which is lighter than VS 2022 and has an easier to use integrated `git`, command line window and lots of other useful features like _auto-save_ and more...

- The project's _CMakeLists.txt_ uses `msvc` and `gcc` on Windows and Linux respectively by default. 
- Using normal CMD to build and run the project seems to work too.

PS: Then, a Shell Script file with the repo's build instructions can be utilized to build and run the project with a single execution on _CMD_:
```
#!/bin/bash
git clone git@github.com:skunkforce/OmniView.git
cd OmniView
git submodule update --init --recursive
mkdir build
cd build
cmake .. -DVCPKG_TARGET_TRIPLET="x64-windows-static"
cmake --build .
cd Debug
./OmniView
```

File: README.md [hunk 1]
```diff
@@ -78,22 +78,47 @@ In order to run OmniView it needs admin-rights right now:
...
+In order to utilize the `MSVC`-Compiler on the Windows Command-Line, you'll have to set a whole bunch of environment variables.
+Thus Build-Tools provides an easy way to load all of them at one. 
+Instead of runnign `cmd` you can just run the `Developer Command Promt` from your Windows Startmenu.
+This starts a Session with all env-variables set accordingly.
+
+Run the following commands in the Developer Command Prompt:
 ```shell
-cmake --build .
+git clone --recurse-submodules git@github.com:skunkforce/OmniView.git
+cd gp_ptp-measurement-access-demon_sw
```
Comment (NP199) [inline README.md:117]: Falscher Path angegeben

**Review by NP199 — COMMENTED** [review]
I could build an run the project on my machine

**Comment by bjoekeldude** [comment]
> * `Microsoft PowerShell` is likely required too.
No.
> * I'd also suggest removing `--config Release` at least for the development stage.
It's supposed to indicate, that this is the correct place to define the build-type

> It's also suitable to use _vs-code_ - which is lighter than VS 2022 and has an easier to use integrated `git`, command line window and lots of other useful features like _auto-save_ and more...
No, this doesn't solve the initial problem. Further it's not about installing `Visual Studio` but about installing the `Visual Studio Build Tools` which is basically a commandline available distribution of `msvc` und the smallest version you can install. 

> * The project's _CMakeLists.txt_ uses `msvc` and `gcc` on Windows and Linux respectively by default.
Yes. The `CMakeLists.txt` should always provide a default option. If someone wants to build with `clang` or `MinGW` they should be able to define their own configuration.

> * Using normal CMD to build and run the project seems to work too.
👍🏻 

> PS: Then, a Shell Script file with the repo's build instructions can be utilized to build and run the project with a single execution on _CMD_:
> 
> ```
> #!/bin/bash
> git clone git@github.com:skunkforce/OmniView.git
> cd OmniView
> git submodule update --init --recursive
> mkdir build
> cd build
> cmake .. -DVCPKG_TARGET_TRIPLET="x64-windows-static"
> cmake --build .
> cd Debug
> ./OmniView
> ```
You are correct, that you can do that, but still the build instructions shall be explained in the `README.md`.

Furthermore I'd argue, that instead of 
```
mkdir build
cd build
cmake ..
```
The idiomatic syntax should be:
```
cmake -S . -B ./build -DVCPKG_TARGET_TRIPLET="x64-windows-static"
```

**Comment by R-Abbasi** [comment]
Installed `Microsoft PowerShell` after encountering the related error (sent a screenshot to the group at the time) then the project has worked properly on Windows.

This can likely be more concise:
```
cmake -S . -B ./build -DVCPKG_TARGET_TRIPLET="x64-windows-static"
cmake --build ./build --config Release
```
like this:
```
cmake -B build -DVCPKG_TARGET_TRIPLET="x64-windows-static"
cmake --build build --config Release
```

**Review by R-Abbasi — APPROVED** [review]

**Pull request merged**